src/stores/TagsStore.ts

```diff
@@ -23,14 +23,20 @@ interface TagsRefreshQueue {
     promise: Promise<void>
 }
 
+interface TagsError {
+    status: string
+    statusText: string
+    url: string
+}
```

Comment:
there is already `RequestError` interface in `Types.ts`

Reply:
Thanks